OpenRA.Game/FileFormats/ImaAdpcmLoader.cs

```diff
@@ -43,7 +43,7 @@ public static class ImaAdpcmLoader
 			1282, 1411, 1552, 1707, 1878, 2066, 2272, 2499, 2749,
 			3024, 3327, 3660, 4026, 4428, 4871, 5358, 5894, 6484,
 			7132, 7845, 8630, 9493, 10442, 11487, 12635, 13899, 15289,
-			16818, 18500, 20350, 22385, 24623, 27086, 29794, 32767 
+			16818, 18500, 20350, 22385, 24623, 27086, 29794, 32767
```

Comment:
I just noticed that this seems to be copy pasta code from `AudLoader.cs`, but I am not going to fix the redundancy here in this pull request.